src/cpp/server/include/server/ServerOptions.hpp

```diff
@@ -86,8 +86,6 @@ class Options : public GeneratedOptions,
       return overlayOptions_[name];
    }
 
-   std::string rsessionExecutable() const;
```

Comment:
It looks like this was removed from pro at some point? I don't see the function being called anywhere.